.github/workflows/cypress_parallel_poc.yaml

```diff
@@ -0,0 +1,210 @@
+name: Cypress Parallelization POC
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+
+
+env:
+  MIX_ENV: test
+  NODE_VERSION: "20"
+  MANTAINERS: '["cdimonaco", "dottorblaster", "janvhs", "nelsonkopliku", "arbulu89","jagabomb","emaksy", "balanza", "gagandeepb"]'
+  RG_TEST_LABEL: regression
+  INTEGRATION_TEST_LABEL: integration
+
+jobs:
+  elixir-deps:
+    name: Elixir ${{ matrix.mix_env }} dependencies
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - mix_env: dev
+          - mix_env: test
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why the need to iterate on both environments>

Reply:
this was copied from Carmine's pr, this file is no longer available :)